scripts/linters/pylint_extensions_test.py

```diff
@@ -3175,21 +3175,13 @@ def test_disallowed_replacements_str(self):
             .checker.config.disallowed_functions_and_replacements_str) = [
                 'datetime.datetime.now=>datetime.datetime.utcnow',
                 'self.assertEquals=>self.assertEqual',
-                'b.next=>python_utils.NEXT',
-                'str=>python_utils.convert_to_bytes or python_utils.UNICODE',
```

Comment:
Let's not remove these tests at it is testing the multiple functionalities of the checker!

Reply:
@DubeySandeep should I test it on other enforcing functions like :
`
  iteritems=>items,
  iterkeys=>keys,
  itervalues=>values,
`

or should I simply revert to the way things were

Reply:
@DubeySandeep I have reverted the test. PTAL

Reply:
Hey, btw, I just realized that we don't need any of  `iteritems=>items`, `iterkeys=>keys`, or `itervalues=>values` since the iter* versions do not exist in Python 3.

Reply:
@vojtechjelinek I have removed them PTAL